web/.gitattributes

```diff
@@ -0,0 +1,61 @@
+# Drupal git normalization
```

Comment:
Is this file needed? Looks like it was duplicated from the directory above `/web`?

Reply:
Ditto - removed this one too